example/request/design/request.yaml

```diff
@@ -157,8 +157,14 @@ spec:
                   gpuNumber: 8
                 - podNumber: 1
                   gpuNumber: 4
-        # Setup gpu, cpu and memory isolation for container runtime.
         spec:
+          # PodPriority should be set proportional to above AffinityGroupPriority,
```

Comment:
Shall we mention the difference of the two priorities? e.g., one is for hived, one is for k8s.

Reply:
sure

---
In reply to: [398985275](https://github.com/microsoft/hivedscheduler/pull/11#discussion_r398985275) [](ancestors = 398985275)